src/skmultiflow/trees/stacked_single_target_regression_hoeffding_tree.py

```diff
@@ -0,0 +1,773 @@
+from operator import attrgetter
+
+import numpy as np
+
+from skmultiflow.core import MultiOutputMixin
+from skmultiflow.trees import RegressionHoeffdingTree
+from skmultiflow.trees import MultiTargetRegressionHoeffdingTree
+from skmultiflow.trees.numeric_attribute_regression_observer_multi_target \
+    import NumericAttributeRegressionObserverMultiTarget
+from skmultiflow.trees.nominal_attribute_regression_observer \
+     import NominalAttributeRegressionObserver
+from skmultiflow.utils.utils import get_dimensions
+from skmultiflow.trees.intra_cluster_variance_reduction_split_criterion \
+     import IntraClusterVarianceReductionSplitCriterion
+
+
+_PERCEPTRON = 'perceptron'
+_ADAPTIVE = 'adaptive'
+
+
+class StackedSingleTargetRegressionHoeffdingTree(
```

Comment:
There is currently no standard to differentiate between classifier and regressors which can be confusing. Maybe we can go with an explicit approach as in `sklearn`, for example in this case:

`StackedSingleTargetRegressionHoeffdingTree` -> `StackedSingleTargetHoeffdingTreeRegressor`

Comments?

Reply:
I think that is a good idea for standardization! Nonetheless, how would these changes impact in the remaining (and existing) methods documentation? Would these changes be gradually implemented?

If you agree, I will create a new commit changing the names in both the algorithm and the corresponding test files (just for SST-HT for now). Let me know what you think! And thank you Jacob for your patience and attention during this PR revision process!

Reply:
This change is indeed tricky for existing methods.

For SST-HT the change is straightforward. However, for existing methods, I think we can go with a deprecation warning for at least a couple of releases before removing the old names, see [here](https://stackoverflow.com/questions/9008444/how-to-warn-about-class-name-deprecation). I will create the issue to perform this change gradually, tentatively during the development of the next release.

Reply:
The stack overflow solution seems to be a good option on the long run. I will change the name of the class and send a new commit.